habitat/utils/visualizations/utils.py

```diff
@@ -84,3 +88,40 @@ def paste_overlapping_image(
     else:
         background_patch[:] = foreground
     return background
+
+
+def images_to_video(
+    images: List[np.ndarray],
+    output_dir: str,
+    video_name: str,
+    fps: int = 10,
+    quality: Optional[float] = 5,
+    **kwargs
+):
+    """Calls imageio to run FFMPEG on a list of images. For more info on
+        parameters, see
```

Comment:
Formatting for docstring here, the current and the next line should be one tab to the left.